lib/modules/datasource/github-releases/cache/cache-base.ts

```diff
@@ -4,6 +4,7 @@ import type {
   GithubGraphqlResponse,
   GithubHttp,
 } from '../../../../util/http/github';
+import type { ChangeLogRelease } from '../../../../workers/repository/update/pr/changelog/types';
```

Comment:
same here, if you really need the interface then it should be moved to global types `lib/types/...`

Looks like we only need `ChangeLogRelease.date`, so you should simply only padd the date?